dev/NumberBox/NumberBox.h

```diff
@@ -69,6 +69,7 @@ class NumberBox :
     void OnNumberBoxLostFocus(winrt::IInspectable const& sender, winrt::RoutedEventArgs const& args);
     void OnNumberBoxScroll(winrt::IInspectable const& sender, winrt::PointerRoutedEventArgs const& args);
     void OnCornerRadiusPropertyChanged(const winrt::DependencyObject& /*sender*/, const winrt::DependencyProperty& /*args*/);
+    void OnIsEnabledChanged(const winrt::IInspectable& /*sender*/, const winrt::DependencyPropertyChangedEventArgs& /*args*/);
```

Comment:
>sender [](start = 57, length = 6)

I think you can just delete the commented out name. In general we have names when they are used, commented out names when they aren't currently used but reasonablly could be, and no name when the variable wont be used.

Reply:
I see. Done. 

While I was at it, I also deleted the commented out names directly above for OnCornerRadiusPropertyChanged (as those won't be used). OnCornerRadiusPropertyChanged, as you might remember, is just used as a "workaround" because the corner radius is not properly being updated for the NumberBox using a converter in XAML markup alone (we discussed that in issue https://github.com/microsoft/microsoft-ui-xaml/pull/2224#discussion_r403373619)).